### PR TITLE
feat : added login and signup

### PR DESCRIPTION
**Pull request by someshtiwarii**
Pages:
added login page
added signup page

### CODE REVIEW

File: src/pages/Login/Login.jsx [hunk 1]
```diff
@@ -1,13 +1,106 @@
...
+
+          <div className="login-heading">
+              <h2>Login</h2>
+          </div>
+
+          <div className="login-input">
+          <form onSubmit={loginHandler}>
+
+              <div className="username-box">
+                  <label for="email">Email:</label>
```
Comment (Sen-442b) [inline src/pages/Login/Login.jsx:51]: You can replace it with react `htmlFor` attribute

File: src/pages/SignUp/SignUp.jsx [hunk 2]
```diff
@@ -1,13 +1,153 @@
...
+import { useState } from "react";
+import { signupService } from "../../services/signupService";
+import { useAuth } from "../../context/authContext";
+import { Link } from "react-router-dom";
 
-const Signup = () => {
+const SignUp = () => {
+  const [signupInput, setSignupInput] = useState({});
+  const { setIsAuth, setToken, navigate } = useAuth();
+  const [hide, setHide] = useState({ pswd: true, confirmpswd: true });
```
Comment (Sen-442b) [inline src/pages/SignUp/SignUp.jsx:11]: You can use  camelCasing for `confirmpswd`

**Review by Sen-442b — APPROVED** [review]
Good Job with the 

- File structure 
- Consistent Variable Naming
- Proper Indentation

Suggestion:- deploy your site  to get   preview of your live site after every new pull request

File: src/pages/SignUp/SignUp.jsx (hunk 2 above)
Comment (someshtiwarii) [reply src/pages/SignUp/SignUp.jsx:11]: Ok i will do that, thankyou for pointing it out

File: src/pages/Login/Login.jsx (hunk 1 above)
Comment (someshtiwarii) [reply src/pages/Login/Login.jsx:51]: I will in the end when i refactor the code, thankyou for reviewing it

**Review by shobit1337 — APPROVED** [review]

**Pull request merged**